src/ontology/mondo-ingest-odk.yaml

```diff
@@ -25,18 +25,20 @@ documentation:
   documentation_system: mkdocs
 components:
   products:
+    - filename: doid.owl
+      source: http://purl.obolibrary.org/obo/doid.owl
+    - filename: gard.owl
+      source: https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl
+    - filename: icd10cm.owl
+      source: https://data.bioontology.org/ontologies/ICD10CM/submissions/21/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
+    - filename: icd10who.owl
+      source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
```

Comment:
FYI: This URL was actually wrong. It was downloading from a release named/tagged 'latest', rather than the actual latest release.

Reply:
I committed this directly to `main` instead along with some other `mondo-ingest-odk.yaml` changes and then rebased this `gard` branch.